sdk/logical/request.go

```diff
@@ -366,6 +366,7 @@ const (
 	HelpOperation                     = "help"
 	AliasLookaheadOperation           = "alias-lookahead"
 	ResolveRoleOperation              = "resolve-role"
+	HeaderOperation                   = "header"
```

Comment:
Would `HeadOperation` not match better the existing values?

```suggestion
	HeadOperation                   = "head"
```